lib/utils.coffee

```diff
@@ -293,7 +295,7 @@ exports.requestAsync = (options, retriesRemaining) ->
 	delete opts.timeout
 
 	requestTime = new Date()
-	p = fetch(url, opts)
+	p = exports.fetch(url, opts)
```

Comment:
That's a good point, one could import `fetch` as `_fetch` which will suggest it's the internal variable.
Not worth a separate PR IMO but can be slipped into one of the upcoming.